src/organisms/navbar/components/UserMenu/UserMenu.tsx

```diff
@@ -13,8 +13,8 @@ export const UserMenu: FC<UserMenuProps> = ({
   const { textShades } = localTheme();
   return (
     <NavbarUserMenu isMenuOpen={isMenuOpen}>
-      {account && <Text size="M-Regular">{shortenAddressTo11Chars(account)}</Text>}
-      {account && <Line length="88px" />}
+      {account && <CopyToClipBoard walletCut text={account} id={account} />}
```

Comment:
This is experimental, i was supposed to add copying when you click the address but thought this looked good and we already use that across the app so it adds to consistency.